cli/flox/doc/flox-containerize.md

```diff
@@ -51,6 +52,14 @@ The produced container however can also run on macOS.
     (default: `./<environment-name>-container.tar`)
     If `<path>` is `-`, writes to `stdout`.
 
+`-t`, `--tag`
+:   Tag the container with `<tag>`
+    (default: `latest`)
+
+`-l`, `--load-into-registry`
+:   Loads the container into `<backend>` without having to pipe
+    Has to be one of `docker` or `podman`.
```

Comment:
nit: idk if it's what a non-Nix user would assume anyways, but I was wondering if it would be good to say we get docker or podman from PATH